packages/babel-plugin-transform-modules-systemjs/test/fixtures/systemjs/export-named-string/output.mjs

```diff
@@ -0,0 +1,15 @@
+System.register([], function (_export, _context) {
+  "use strict";
+
+  var foo, bar;
+
+  _export({
+    foo: void 0,
+    bar: void 0
+  });
```

Comment:
This output is incorrect, it should be
```js
_export({
    "default exports": foo,
    bar: void 0
  });
```
However it is a regression introduced in https://github.com/babel/babel/pull/9589/files#diff-a0b4df402be122c3213c96111b909761. I will address it in a separate PR.

Reply:
Resovled in #12110.